pkg/commands/start_test.go

```diff
@@ -106,6 +125,10 @@ func dnsLookup(t *testing.T, resolverPort, resolverProtocol, domain string) ([]n
 	return resolver.LookupIP(ctx, "ip4", domain)
 }
 
+func randomPort() string {
+	return fmt.Sprint(rand.Intn(32767-1024) + 1024)
```

Comment:
What you have is acceptable; but consider using:
```suggestion
	return fmt.Sprint(rand.Intn(32767) + 32768)
```
(i.e. some number between 32768 to 65535, inclusive)